SimplifiedTests/OPDS2LibraryRegistryFeed.json

```diff
@@ -0,0 +1,347 @@
+{
+  "metadata":{
+    "title":"Libraries near 40.7769,-73.9813"
+  },
+  "links":[
+    {
+      "title":"Libraries near 40.7769,-73.9813",
+      "rel":"self",
+      "href":"https://libraryregistry.librarysimplified.org/nearby?location=40.7769,-73.9813",
+      "type":"application/opds+json;profile=http://librarysimplified.org/rel/profile/directory"
+    }
+  ],
+  "catalogs":[
+    {
+      "metadata":{
+        "title":"Boston Public Library",
+        "identifier":"urn:uuid:edef2358-9f6a-4ce6-b64f-9b351ec68ac4",
+        "modified":"2021-01-12T17:48:06Z",
+        "schema:distance":"14 km",
```

Comment:
@leonardr would it be possible to put the unit of measure in a separate field? The client will have to do conversions between km and miles depending on the user's locale settings, so it would save us some parsing. If it's always going to be in km I suppose we could even omit the `km` part, but that seems less future-proof.

Reply:
This comes from the definition of https://schema.org/Distance:

```
Properties that take Distances as values are of the form '<Number> <Length unit of measure>'. E.g., '7 ft'.
```

So we would need to make it a custom field rather than schema:distance. I would prefer to leave things as is unless parsing out the unit of measurement is a big problem.